docs/sources/operators-guide/configure/configuring-tsdb-block-upload.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Configuring TSDB block upload"
+menuTitle: "Configuring TSDB block upload"
+description: "Learn how to configure Grafana Mimir to enable TSDB block upload"
+weight: 120
+---
+
+# Configuring TSDB block upload
+
+Grafana Mimir supports uploading of historic TSDB blocks, sourced from f.ex. Prometheus, Thanos, or
```

Comment:
Let's not mention Thanos here, as we reject ALL thanos blocks right now (due to them having external labels).

Reply:
Done.